docs/source/index.rst

```diff
@@ -141,17 +141,17 @@ clicking the "cite this repository" button on the right sidebar (which will gene
 a citation in your preferred format) or by copying the following BibTeX entry:
 ```
```

Comment:
maybe here it might be better to do:

```
.. code:: text
  @article{Landry_XGI_2023,
   author = {Landry, Nicholas W. and Lucas, Maxime and Iacopini, Iacopo and Petri, Giovanni and Schwarze, Alice and Patania, Alice and Torres, Leo},
   title = {{XGI: A Python package for higher-order interaction networks}},
   doi = {10.21105/joss.05162},
   journal = {Journal of Open Source Software},
   publisher = {The Open Journal},
   year = {2023},
   month = may,
   volume = {8},
   number = {85},
   pages = {5162},
   url = {https://joss.theoj.org/papers/10.21105/joss.05162},
   }
```

Reply:
Good point! Fixed.